gems/pending/disk/modules/AzureBlobDisk.rb

```diff
@@ -72,7 +72,7 @@ def blob_read(start_byte, length)
     ret = @storage_acct.get_blob_raw(@container, @blob, key, options)
 
     content_md5  = ret.headers[:content_md5].unpack("m0").first.unpack("H*").first
-    returned_md5 = Digest::MD5.hexdigest(ret.body)
+    returned_md5 = Rails.application.config.digest_class.hexdigest(ret.body)
```

Comment:
@woot4moo The various modules under `gems/pending` provide functionality that can be used outside of the ManageIQ Rails application. The intent is to move this code out into independent GEMs which could then be used by MIQ as well as other applications (hence the `gems/pending`).

As for the MD5 check being redundant, I think we're in agreement there. The check should be removed.